dali/base/dasds.cpp

```diff
@@ -6073,7 +6086,15 @@ CCovenSDSManager::CCovenSDSManager(ICoven &_coven, IPropertyTree &_config, const
     doTimeComparison = false;
     if (config.getPropBool("@lightweightCoalesce", true))
         coalesce.setown(new CLightCoalesceThread(config, iStoreHelper));
-    size_t extCacheSize = config.getPropInt("@extCacheSizeMB", defaultExtCacheSizeMB) * 0x100000;
+
+    // if enabled (default), ensure the ext cache size is bigger than the pending delta writer cache
+    // to ensure that when the ext cache is full, there are transaction items to flush that aren't still
+    // pending on write.
+    unsigned extCacheSizeMB = config.getPropInt("@extCacheSizeMB", defaultExtCacheSizeMB);
+    unsigned deltaTransactionMaxMemMB = config.getPropInt("@deltaTransactionMaxMemMB", defaultDeltaMemMaxMB);
+    if (extCacheSizeMB && deltaTransactionMaxMemMB && (extCacheSizeMB < deltaTransactionMaxMemMB))
```

Comment:
Should this be <= rather than <?

Reply:
yes, it should, I'll change and squash that minor change in.